scripts/circleci-jobs/openshift3-integration-tests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+
+DOCKERHUB_USER = os.getenv('DOCKERHUB_USER')
```

Comment:
I would use capital letters for maybe some of the consts you want to shout about, but not for all vars. wdyt? makes it a bit hard to read